app/templates/components/header.hbs

```diff
@@ -1,17 +1,43 @@
 <PaperToolbar class="header">
-    <PaperToolbarTools>
-    <PaperButton @disabled={{false}} @raised={{false}} @fab={{false}} @bubbles={{true}} @href={{href-to "index"}}><img class="pilas-bloques-logo" src='imagenes/logo.png'></PaperButton>
-    <PaperButton @disabled={{false}} @raised={{false}} @fab={{false}} @bubbles={{true}} @href={{href-to "libros.verLibro" 1}}>{{paper-icon "menu_book" class="menu-button-icon"}}<span class="menu-button-typography">Libro 1</span></PaperButton>
-    <PaperButton @disabled={{false}} @raised={{false}} @fab={{false}} @bubbles={{true}} @href={{href-to "libros.verLibro" 2}}>{{paper-icon "menu_book" class="menu-button-icon"}}<span class="menu-button-typography">Libro 2</span></PaperButton>
-    <span class="flex"></span>
-    {{#if model.titulo}}
-      <div class="header-title">{{model.titulo}}</div>
-      <span class="flex"></span>
-      <span class="flex"></span>
+  <PaperToolbarTools>
+
+     <a class="pilas-bloques-logo" href="#"><img src='imagenes/logo.png'></a>
+
+    {{#unless model}}
+
+      {{!-- primer ciclo href button --}}
+      <PaperButton 
+        @disabled={{false}} 
+        @raised={{false}} 
+        @fab={{false}} 
+        @bubbles={{true}} 
+        @href={{href-to "libros.verLibro" 1}}>
+          {{paper-icon "menu_book" class="menu-button-icon"}}<span class="menu-button-typography">Primer Ciclo</span>
+      </PaperButton>
```

Comment:
Se podría abstraer un `<LibroButton/>` esta estructura se repite y se podría encapsular el `href-to "libros.verLibro" `.

Reply:
me gusta!